app/services/analytics_events.rb

```diff
@@ -1116,6 +1116,39 @@ def idv_personal_key_submitted(
     )
   end
 
+  # Multi factor auth add new phone
+  # @param [Boolean] success
+  # @param [Hash] errors
+  # @param [String] otp_delivery_preference
+  # @param [String] area_code
+  # @param [String] carrier
+  # @param [String] country_code
+  # @param [String] phone_type
+  # @param [Hash] types
+  def multi_factor_auth_add_phone_setup(success:,
+                                        errors:,
+                                        otp_delivery_preference:,
+                                        area_code:,
+                                        carrier:,
+                                        country_code:,
+                                        phone_type:,
+                                        types:,
+                                        **extra)
+
+    track_event(
+      'Multi-Factor Authentication: Add new phone setup',
```

Comment:
Should we use the existing `multi_factor_auth_setup` method rather than creating one specific for phones? We use it for the other MFAs.

Reply:
So that would happen at the OTP confirmation which I don't think we need to do. I think instead maybe we could use multi_factor_auth_phone_setup which we do on just entering the phone number again.

Reply:
I think it's fine to re-use that one, yeah, since both events are upon trying to create a phone (before verification)